packages/arb-token-bridge-ui/src/hooks/useArbQueryParams.tsx

```diff
@@ -97,6 +108,50 @@ export const AmountQueryParam = {
   }
 }
 
+// Parse chainId to ChainQueryParam or ChainId for orbit chain
+function encodeChainQueryParam(chainId: number | null | undefined) {
+  if (!chainId) {
+    return undefined
+  }
+
+  try {
+    const chain = getChainQueryParamForChain(chainId)
+    return chain.toString()
+  } catch (e) {
+    return undefined
```

Comment:
Should we do something special in this case? Will the sanitization function catch this?